kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/modem/quectel/ec25/QuectelEC25ConfigGenerator.java

```diff
@@ -0,0 +1,148 @@
+/*******************************************************************************
```

Comment:
Please see the comments for the other class similar to this one

Reply:
ok